src/System.Runtime.Extensions/tests/System/UnloadingAndProcessExitTests.netcoreapp1.1.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+namespace System.Tests
+{
+    public class UnloadingAndProcessExitTests : RemoteExecutorTestBase
+    {
+        [Fact]
+        public void UnloadingEventMustHappenBeforeProcessExitEvent()
+        {
+            string fileName = GetTestFilePath();
+
+            File.WriteAllText(fileName, String.Empty);
```

Comment:
nit: CoreFX coding guidelines prefer using `string` rather than `String` since `string` is intrinsic to the language